### ISSUE TITLE
Ability to select language mode in command palette itself

### ISSUE THREAD

**Issue by virajvchavan** [microsoft/vscode#85296]
Currently, to select/change the language mode of a file, we need to click on the button at the bottom, which invokes the 'Language Mode Changer'.

I'd be great if we can select a language mode in the generic command palette itself.
(Like how it is in Sublime Text 3)
Usage example:
1. I type `Shift+Ctl+P`, the command palette opens up
2. I start typing javascript, the suggestions include: (`Select Language Mode: JavaScript`)
3. I select it, the language mode is changed.

**Comment by bpasero** [comment]
@virajvchavan well you can do this to bring up the picker, isn't that enough?

![image](https://user-images.githubusercontent.com/900690/70223189-8c2d8180-174b-11ea-9dd0-cbf365014aa0.png)

**Comment by virajvchavan** [comment]
> @virajvchavan well you can do this to bring up the picker, isn't that enough?
It does seem enough for a user if he knows about this and is used to this. 

For a new user, especially someone who comes from using Sublime Text, it will be intuitive to do this in one go from the generic command palette.

I created this issue because as a previous Sublime user(which has what I have described), to change the language mode I started typing 'javascript' in the command palette directly, and it did not give me an option to do so. Maybe there are some number of users who feel the same.
